server/app.js

```diff
@@ -17,6 +21,22 @@ app.use(compression({ threshold: 0 }));
 mongoose.Promise = bluebird;
 mongoose.connect(process.env.MONGO_URI);
 
+const logDirectory = path.join(__dirname, 'log');
```

Comment:
pm2 controls the output logs from the node processes. There is no need to rotate the logs or to couple the app to the file system